meinberlin/apps/budgeting/phases.py

```diff
@@ -77,6 +77,21 @@ class RatingPhase(phases.PhaseContent):
     features = {"rate": (models.Proposal,)}
 
 
+class BudgetingTwoSupportPhase(phases.PhaseContent):
+    app = apps.Config.label
+    phase = "support_budgeting_two"
```

Comment:
I wasn't sure here, is this used anywhere? I just picked a random name

Reply:
You can pick anything, thats fine, it is used as registry name later